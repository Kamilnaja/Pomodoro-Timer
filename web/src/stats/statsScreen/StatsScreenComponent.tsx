import { Container } from 'react-bootstrap';
import StatsContainer from '../stats/containers/StatsContainer';
import StatsWithTagsContainer from '../statsWithTags/containers/StatsWithTagsContainer';

export const StatsScreenComponent = () => (
  <Container>
    <h2>Statistics</h2>
    <StatsContainer />
    <StatsWithTagsContainer />
  </Container>
);
