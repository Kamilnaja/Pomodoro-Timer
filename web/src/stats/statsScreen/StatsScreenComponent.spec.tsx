import { shallow, ShallowWrapper } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { StatsScreenComponent } from './StatsScreenComponent';

describe('StatsScreenComponent', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<StatsScreenComponent />);
  });

  it('should match snapshot', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
