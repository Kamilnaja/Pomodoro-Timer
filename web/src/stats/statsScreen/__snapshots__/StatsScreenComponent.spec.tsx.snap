// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsScreenComponent should match snapshot 1`] = `
<Container
  fluid={false}
>
  <h2>
    Statistics
  </h2>
  <Connect(StatsContainer) />
  <Connect(StatsWithTagsContainer) />
</Container>
`;
